tests/unittests/OperatorTest.cpp

```diff
@@ -66,6 +67,303 @@ static Tensor createTensorConditionallyQuantized(ElemKind T,
   return isQuantizedElemKind(T) ? Tensor(T, dims, 1.0, 0) : Tensor(T, dims);
 }
 
+template <typename DataType>
+glow::Handle<DataType>
+whereHelper(glow::PlaceholderBindings &bindings, glow::Module &mod,
+            glow::Function *F, glow::ExecutionEngine &EE, ElemKind DTy,
+            llvm::ArrayRef<DataType> xValues, llvm::ArrayRef<DataType> yValues,
+            llvm::ArrayRef<bool> cValues, llvm::ArrayRef<size_t> xDims,
+            llvm::ArrayRef<size_t> yDims, llvm::ArrayRef<size_t> cDims) {
+  Node *cond = createPlaceholderConditionallyQuantized(mod, ElemKind::BoolTy,
+                                                       cDims, "cond", false);
+  Node *X = createPlaceholderConditionallyQuantized(mod, DTy, xDims, "X",
+                                                    DTy != ElemKind::FloatTy);
+
+  Node *Y = createPlaceholderConditionallyQuantized(mod, DTy, yDims, "Y",
+                                                    DTy != ElemKind::FloatTy);
+
+  bindings.allocate(llvm::dyn_cast<Placeholder>(cond))->getHandle<bool>() =
+      cValues;
+
+  bindings.allocate(llvm::dyn_cast<Placeholder>(X))->getHandle<DataType>() =
+      xValues;
+
+  bindings.allocate(llvm::dyn_cast<Placeholder>(Y))->getHandle<DataType>() =
+      yValues;
+
+  EXPECT_EQ(xDims.size(), yDims.size());
+  EXPECT_EQ(yDims.size(), cDims.size());
+
+  llvm::SmallVector<size_t, 4> targetDims;
+  bool broadcastInput = false;
+  bool broadcastCond = false;
+  if (xDims != yDims) {
```

Comment:
It seems like much of this logic (e.g. deciding whether to broadcast/performing the broadcast for `x`/`y`/`c`) is shared both here and in the ONNXModelLoader. I think it might make more sense to extract all of this into a `Function::createWhere()`, so that we can be sure the logic is used both in OperatorTest and ONNXModelLoader. This might include moving `computeMultidirectionalBroadcast()` to somewhere accessible by both Graph and Importers. WDYT?

Reply:
I agree. In original implementation I had some, limited, support for broadcast right in create function. For this version I followed what was currently there, but it does lead to duplication of code and pushing handling of this to the user of the create function.
Would refactor make sense as part of this MR, or a separate one?

Reply:
I'd prefer this one, perhaps as a separate commit so it's easier to review. But if you'd like to file a follow up it's NBD.

Reply:
Sure I can add it to this one, after it goes through internal flow.

Reply:
@jfix71 checkout https://github.com/pytorch/glow/pull/3170